pkgs/tools/security/sbomnix/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, coreutils
+, curl
+, gnugrep
+, gnused
+, gzip
+, graphviz
+, nix
+, reuse
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "sbomnix";
+  version = "1.4.5";
+
+  src = fetchFromGitHub {
+    owner = "tiiuae";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-RxDFxVGivVBw2uhtzf231Q2HHTBFKSqGrknr2Es/ygM=";
+  };
+
+  makeWrapperArgs = [
+    "--prefix PATH : ${lib.makeBinPath [ coreutils curl gnugrep gnused gzip graphviz nix ]}"
+  ];
+
+  propagatedBuildInputs = [
+    reuse
+    python3Packages.colorlog
+    python3Packages.graphviz
+    python3Packages.numpy
+    python3Packages.packageurl-python
+    python3Packages.pandas
+    python3Packages.requests
+    python3Packages.tabulate
+  ];
+
+  pythonImportsCheck = [ "sbomnix" ];
+
+  meta = with lib; {
+    description = "Generate SBOMs for nix targets";
+    homepage = "https://github.com/tiiuae/sbomnix";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ henrirosten ];
```

```suggestion
    maintainers = with maintainers; [ henrirosten jk ];
```

I'd be happy to maintain this with you if you'd like

---

Thanks! Added you as maintainer in 0f1b2e7747bb9dea653ca76d107028bc25450b22